src/index.ts

```diff
@@ -137,7 +137,7 @@ export class RingCentralCallControl extends EventEmitter {
     if (message.event.indexOf('/telephony/sessions') === -1) {
       return;
     }
-    const { eventTime, telephonySessionId, sessionId, ...newData } = message.body;
+    const { telephonySessionId, ...newData } = message.body;
```

Comment:
I guess you want to use eventTime as creationTime. So we can set session.creationTime = eventTime if creationTime is not existed.